components/organisation-unit-tree/src/__stories__/custom-node-label.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { OrganisationUnitTree } from '../index.js'
+
+export const CustomNodeLabel = () => (
+    <OrganisationUnitTree
+        name="Root org unit"
+        roots="A0000000000"
+        initiallyExpanded={['/A0000000000/A0000000001']}
+        renderNodeLabel={data => {
+            if (data.node.path !== '/A0000000000/A0000000001') {
+                return OrganisationUnitTree.defaultProps.renderNodeLabel(data)
+            }
+
+            return OrganisationUnitTree.defaultProps.renderNodeLabel({
+                ...data,
+                label: <span>--- {data.node.displayName}</span>,
+            })
+        }}
```

Comment:
Since using default props in this way is not a typical react pattern, I propose we expose `Label` as `OrganisationUnitLabel` in order to allow library consumers to do the following:

```js
        renderNodeLabel={props => {
            if (props.node.path !== '/A0000000000/A0000000001') {
                return <Label {...props}>{props.label}</Label>
            }

            return (
              <Label {...props}>
                Custom label for {props.node.displayName}
              </Label>
            )
        }}
```

Reply:
Alternatively we could make `renderNodeLabel` less flexible and use the return value as the `children` prop of the label:

```js
        renderNodeLabel={node => {
            if (node.path !== '/A0000000000/A0000000001') {
                return node.displayName
            }

            return (
              <strong>
                Custom label for <span>{props.node.displayName}</span>
              </strong>
            )
        }}
```

Reply:
I really like the idea of making it less flexible! WIll adapt that version